spec/lib/congress_forms_spec.rb

```diff
@@ -0,0 +1,89 @@
+require "rails_helper"
+
+describe CongressForms do
+  describe CongressForms::Form do
+    let(:form) {
+      CongressForms::Form.new("C000880", [
+        { "value" => "$NAME_FIRST" },
+        { "value" => "$NAME_LAST" },
+        { "value" => "$ADDRESS_STATE", "options_hash" => {
+            "CALIFORNIA" => "CA",
+            "NEW YORK" => "NY"
+        } }
+      ])
+    }
+
+    let(:input) { {
+      "$NAME_FIRST" => "Willow",
```

Comment:
I'm not sure what this is for?

Reply:
Yep! Everything under `/spec` and `/features` is for testing.

In the past we've put browser tests in `/features` and used [Cucumber]https://cucumber.io/, but going forward we're putting all our testing in `/spec` (including browser tests) and dropping that tool.